src/kodeagent/__init__.py

```
@@ -30,6 +30,69 @@
+
+def _add_work_dir_to_codeact_prompt(agent: 'CodeActAgent') -> None:
+    """Append work-dir guidance to the CodeActAgent system prompt."""
+    work_dir = _get_codeact_work_dir(agent)
+    if not work_dir:
+        return
+    instruction = _CODEACT_WORK_DIR_INSTRUCTION.format(work_dir=work_dir)
+    for attr in ('system_prompt', 'persona', '_system_prompt', 'system_prompt_template', 'prompt'):
+        try:
+            current = getattr(agent, attr)
+        except Exception:
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)